shell/initialize/entry-helpers.js

```diff
@@ -123,30 +122,29 @@ const checkForErrors = (app) => {
  */
 function callMiddleware(Components, context) {
   let midd = [];
-  let unknownMiddleware = false;
 
   Components.forEach((Component) => {
     if (Component.options.middleware) {
       midd = midd.concat(Component.options.middleware);
     }
   });
 
+  midd = midd.filter((middleware) => {
```

Comment:
Sorry for asking, but do we really need this?
Which cases did you cross or had in mind that would need this as a fallback?

Reply:
Extensions with middleware will cause the app to fail here when we attempt to lookup and execute the non-existant middleware by name.